examples/configuration-governance.v1.test.js

```diff
@@ -0,0 +1,400 @@
+/**
+* @jest-environment node
+*/
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const ConfigurationGovernanceV1 = require('../dist/configuration-governance/v1');
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+const authHelper = require('../test/resources/auth-helper.js');
+
+// Location of our config file.
+const configFile = 'configuration_governance.env';
+
+const describe = authHelper.prepareTests(configFile);
+
+// Save original console.log and console.warn
+const originalLog = console.log
+const originalWarn = console.warn
+
+// Mocks for console.log and console.warn
+const consoleLogMock = jest.spyOn(console, 'log');
+const consoleWarnMock = jest.spyOn(console, 'warn');
+
+describe('ConfigurationGovernanceV1', () => {
+
+  // begin-common
+
+  const configurationGovernanceService = ConfigurationGovernanceV1.newInstance({});
+
+  // end-common
+
+  // Globlal variables to hold link values
+  let attachmentEtag
+  let attachmentId
+  let ruleEtag
+  let ruleId
+
+  const config = readExternalSources(ConfigurationGovernanceV1.DEFAULT_SERVICE_NAME);
+  
+  // Additional configuration settings
+  let testLabel = 'NodeSDKExamples';
+  let accountId = config.accountId;
+  let serviceName = config.exampleServiceName;
+  let enterpriseScopeId = config.enterpriseScopeId;
+  let subacctScopeId = config.subacctScopeId;
+
+  beforeAll(async done => {
+    // Clean any existing test rules before we start the actual tests.
+    await cleanRules(testLabel, done);
+  });
+
+  test('createRules request example', done => {
+
+    consoleLogMock.mockImplementation(output => {
+      let responseBody = JSON.parse(output);
+      ruleId = responseBody.rules[0].rule.rule_id;
+      done();
+    });
+    consoleWarnMock.mockImplementation(output => {
+      done(output);
+    });
+
+    // begin-create_rules
+
+    const params = {
+      rules: [{ request_id: '3cebc877-58e7-44a5-a292-32114fa73558', rule: { account_id: accountId, name: 'Disable public access', description: 'Ensure that public access to account resources is disabled.', labels: ['Access', 'IAM'], target: { service_name: serviceName, resource_kind: 'service' }, required_config: { description: 'Public access check', and: [{ property: 'public_access_enabled', operator: 'is_false' }] }, enforcement_actions: [{ action: 'disallow' }, { action: 'audit_log' }] } }],
```

Comment:
We should look into "pretty printing" or better yet "lint fixing" lines like these so that they match JS style guidelines.

Just a thought, not necessarily required for this PR

Reply:
BTW, we have a similar situation in python as well I think.